dashboard/config/routes.rb

```diff
@@ -115,6 +115,12 @@ module OPS
     end
   end
 
+  resources :projects_lists, path: '/projects-lists/', only: [:index] do
+    collection do
+      get "/:section_id", to: 'projects_lists#index'
+    end
+  end
```

Comment:
Since this is just one route, you could also define it on a single line as 
```ruby
get '/projects-lists/:section_id', to: 'projects_lists#index'
```